msticpy/data/queries/kql_sent_lxauditd.yaml

```diff
@@ -0,0 +1,56 @@
+metadata:
+  version: 1
+  description: Linux Auditd Queries
+  data_environments: [LogAnalytics]
+  data_families: [LinuxAudit]
+  tags: ['linux', 'audit', "auditd"]
+defaults:
+  metadata:
+    data_source: 'linux_auditd'
+  parameters:
+      table:
+        description: Table name
+        type: str
+        default: 'AuditLog_CL'
+      start:
+        description: Query start time
+        type: datetime
+      end:
+        description: Query end time
+        type: datetime
+      add_query_items:
+        description: Additional query clauses
+        type: str
+        default: ''
+      subscription_filter:
+        description: Optional subscription/tenant filter expression
+        type: str
+        default: 'true'
+      host_name:
+        description: Hostname to query for
+        type: str
+        default: ''
+sources:
+  auditd_all:
+    description: Extract all audit messages grouped by mssg_id
+    args:
+      query: '
+        {table}
+        | where {subscription_filter}
+        | where TimeGenerated >= datetime({start})
+        | where TimeGenerated <= datetime({end})
+        | where Computer has "{host_name}"
+        // extract the message parts from the RawData field
+        | extend mssg_parts = extract_all(@"type=(?P<type>[^\s]+)\s+msg=audit\((?P<mssg_id>[^)]+)\):\s+(?P<mssg>[^\r]+)\r?",
```

Comment:
Comments don't work in these queries - its concats the query to one string so anything after the first comment line gets commented out.

Reply:
Looks like you fixed this.